example-application/api-under-test.js

```diff
@@ -82,20 +81,22 @@ const defineRoutes = (expressApp) => {
   expressApp.use("/order", router);
 
   expressApp.use(async (err, req, res, next) => {
-    console.log(err);
-    if (process.env.SEND_MAILS === "true") {
-      // important notification logic here
-      await mailer.send("Error", err.message, "admin@app.com");
-
-      // Other important notification logic here
+    if (typeof err === "object") {
+      if (err.isTrusted === undefined || err.isTrusted === null) {
+        err.isTrusted = true; //Error during a specific request is usually not catastrophic and should not lead to process exit
+      }
     }
```

Comment:
```suggestion
      err?.isTrusted = true; //Error during a specific request is usually not catastrophic and should not lead to process exit
```

Reply:
Can I do optional chaining with an assignment? 🤔

Reply:
you're right, my mistake..
I just toy with js a lil bit and you can also do:
```
typeof err === "object" && !err.isTrusted && (err.isTrusted = true)
```

But you can resolve it :D

Reply:
you're right, my mistake..